RockPaperScissors.iml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be ignored as well. Let us know if you need help with getting git to ignore the file after it's been committed like it has been here.